app/routes/products/category.tsx

```diff
@@ -0,0 +1,5 @@
+const CategoryRoute = () => {
```

Comment:
What's the difference between this route and the index one?

Reply:
I forgot to include the category id in the file name. The idea is to use index for all products and use $categoryId for filtering by category